### ISSUE TITLE
Azure Devops Pipeline Json super linter Failure - fatal: not a git repository (or any parent up to mount point /tmp) Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

### ISSUE THREAD

**Issue by arun6842** [github/super-linter#3448]
## Describe the bug

Unable to execute json lint using super-linter container, followed the below doc. But still getting error.
https://blog.tyang.org/2020/06/27/use-github-super-linter-in-azure-pipelines/

Command used:
```
docker run -e USE_FIND_ALGORITHM=true -e RUN_LOCAL=true -e GIT_DISCOVERY_ACROSS_FILESYSTEM=true -e VALIDATE_ALL_CODEBASE=true -e VALIDATE_JSON=true -v $(System.DefaultWorkingDirectory):/tmp/lint github/super-linter
```

Error:

$(System.DefaultWorkingDirectory) is a git repository, .git folder with all required configs exists but still super-linter isn't able to identify it as a git repository.

<img width="819" alt="image" src="https://user-images.githubusercontent.com/115564231/195113496-008ec1e2-39fa-45be-90f4-13d70802670a.png">

```
fatal: not a git repository (or any parent up to mount point /tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
```

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/115564231/195112951-33c2865b-c33f-48f1-aba4-5af716bd0935.png">

## Expected behavior

If there is a json error super linter container should detect and fail the job with valid error message. Works fine locally.

<img width="819" alt="image" src="https://user-images.githubusercontent.com/115564231/195115259-743e1e5b-a4d1-495b-bce4-2bf53e71dccd.png">

## Steps to Reproduce

1. Click on '....'
1. Scroll down to '....'
1. See error

## Logs

Attached log file
[log.txt](https://github.com/github/super-linter/files/9756699/log.txt)
